omod/src/main/webapp/viewLogEntries.jsp

```diff
@@ -48,7 +48,7 @@
 					</select>
 				</td>
 				<th><spring:message code="idgen.identifierContains"/>:</th>
-				<td width="100%"><input type="text" name="identifier" value="${identifier}"/></td>
+				<td width="100%"><input type="text" name="identifier" value="${ui.encodeHtmlContent(identifier)}"/></td>

```

Comment:
Shouldn't we use encodeHtmlAttribute here?

Reply:
Yes, I've made the changes now. Thank you for pointing it out!